semainda/fast-food-api
# Ft add category #160787273

### What does this PR do?
Have CRUD on menu categories endpoint in FAST FOOD API

### Description of Task to be completed?
Have the following endpoints working
GET /api/v2/menu/categories
POST /api/v2/menu/categories
GET /api/v2/menu/categories/<cat_id>
PUT /api/v2/menu/categories/<cat_id>
DELETE /api/v2/menu/categories/<cat_id>

### How should this be manually tested?
After cloning the repo, cd into the api folder then RUN python run.py
- You should be prompted with Key Errors, export the ENV_CONFIG and DATABASE_URL for your environment.
- Using the POstman test every endpoint mentioned above with body header application/json

### Any background context you want to provide?
The implementation is ongoing to add more endpoints

### What are the relevant pivotal tracker stories?
#160787273
